src/components/form/day-picker/index.tsx

```diff
@@ -62,6 +62,8 @@ export interface DayPickerProps extends InputComponent, DayPickerSingleProps {
   title?: string;
   value?: Date;
   wrapperProps?: any;
+  // eslint-disable-next-line typescript-sort-keys/interface
+  preventClosingOnDaySelect?: boolean;
```

Comment:
just put `preventClosingOnDaySelect` in right position, to follow alphabetical  orded